### DISCUSSION TITLE
TrackingFIelds in the Meeting model

### DISCUSSION THREAD

**Discussion by SilverrainSPB** [Jericho/ZoomNet#289] [Q&A] [answered]
Hello, everyone!
I wanted to use the Tracking Fields functionality in my application. And I tried ZoomClient.Meetings.GetAsync(long id) method to read the tracking fields for related meeting, but I was a bit frustrated. Is there the reason why the Meeting model does not contain property for tracking fields?

I have seen that tracking fields can be included when creating a meeting in the ZoomClient.Meetings.CreateRecurringMeetingAsync and CreateScheduledMeetingAsync methods. Are they not considered added by ZoomNet?

**Comment by Jericho** [comment, marked as answer]
>  Is there the reason why the Meeting model does not contain property for tracking fields?

The ability to retrieve a meeting was added to ZoomNet in March 2019 and the `Meeting` model was created at that time based on the documentation which was available at that time. Therefore, tracking fields are not part of the Meeting model either because they were not documented at that time or they were overlooked. 

> Are they not considered added by ZoomNet?

I'm afraid I don't understand what you mean. Can you clarify your question?

**Reply by SilverrainSPB** [reply]
Nevermind, thank you for answer:)
